cli/config/config.go

```diff
@@ -150,3 +155,24 @@ func (m *Metadata) ExpandEnv() {
 		m.Tags[i] = os.ExpandEnv(v)
 	}
 }
+
+// SyncCapabilities sync capabilities if no settings
+func (p *Project) SyncCapabilities() {
+	for i, s := range p.Suites {
+		if len(s.Settings.BrowserName) == 0 {
```

Comment:
1. It's more canonical to do `s.Settings.BrowserName == ""`
2. I think I know what you are trying to use this check for, but I can have settings that don't define the browser name and instead change other values. The "browser name check" is not a reliable way to check whether settings have been specified at all.